test/test_change_stream.py

```diff
@@ -121,6 +122,7 @@ def kill_change_stream_cursor(self, change_stream):
 
 
 class APITestsMixin(object):
```

Comment:
I think this file is improperly using inheritance. Could you open a tech debt ticket so we can fix this at some point?

Reply:
I addressed the code review comments, I'll handle the mypy/JIRA tickets tomorrow

Reply:
https://jira.mongodb.org/browse/PYTHON-3106